content/concepts/basics/04-search.md

```diff
@@ -7,11 +7,41 @@ metaDescription: "sensenet Search"
 # Search
 https://community.sensenet.com/docs/search-engine/
 https://wiki.sensenet.com/query_syntax.html
-# Full text search
+
+In sensenet one of the most important technical aspects is how we index content items and how we search for them. Especially because sensenet is a query based system which means every content, every list you see on the UI is there because a query found it. In this article we describe the search module and how developers can customize it.
+
 # Metadata indexing, searching by fields
-https://wiki.sensenet.com/Field_Indexing.html
+Making full index of content repository content is one of the most important tasks of the installation procedure of the sensenet Content Repository. This feature is used frequently during development too. The tool that can be used to create a full index is the Index Populator.
+Index Populator is a standalone tool that has two files:
+
+- IndexPopulator.exe
+- IndexPopulator.exe.config
+
+For every content [link] the field values can be indexed so that when searched for a field value the corresponding content will appear in the result set. It is also possible to search in fields by explicitely defining the field whose values are to be searched within a query. The way a specific field of a content is indexed is defined in the CTD Field Definition.
```

Comment:
```suggestion
For every content [link] the field values can be indexed so that when searched for a field value the corresponding content will appear in the result set. It is also possible to search in fields by explicitly defining the field whose values are to be searched within a query. The way a specific field of a content is indexed is defined in the CTD Field Definition.
```